src/styles/calendar.scss

```diff
@@ -0,0 +1,22 @@
+.calendar {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  box-sizing: border-box;
+  flex-wrap: wrap;
+  gap: $gap-pix;
+  width: $calendar-width;
+  padding: $base-pix * 2;
+}
+
+@for $last-day from 28 through 31 {
+  .calendar--month-length_#{$last-day}:nth-child(n + #{$last-day} + 1) {
```

Comment:
this loop doesn't work on demo, check it

Reply:
fixed